File: sync/src/main/java/com/inceptionnotes/sync/events/IdentifyEvent.java
package com.inceptionnotes.sync.events;

import com.inceptionnotes.sync.Client;

/**
 * Created by jacob on 1/30/18.
 */

public class IdentifyEvent extends Event {

    public String me;
    public String client;

    @Override
    public void got(Client client) {
        client.identify(me, this.client);
    }
}

File: sync/src/main/java/com/inceptionnotes/sync/events/IdentifyEvent.kt
package com.inceptionnotes.sync.events

import com.inceptionnotes.sync.world.Client

/**
 * Created by jacob on 1/30/18.
 */

class IdentifyEvent : Event {

    var me: String? = null
    var client: String? = null

    override fun got(client: Client) {
        client.identify(me, this.client)
    }
}
